### PR TITLE
update vscode workspace folders

### PR DESCRIPTION
**Pull request by svenefftinge**
## Description
Updates the vs code workspace to include all leeway components that contain either a `go.mod` or a `package.json` file alphabetically sorted.

The reason for this change is that there is currently no ordering nor strategy to what is included and what is not. Just including all components might be ok if it doesn't pay a too big toll on the performance of the workspace/language servers.

## Related Issue(s)

Fixes #

## How to test

start a workspace on this PR and check the VS Code experience

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Comment by akosyakov** [comment]
It works, but I'm worse person to ask. My flow: collapse everything, open windows for individual components.

**Comment by easyCZ** [comment]
Is there a reason to add *every single one*?

**Comment by svenefftinge** [comment]
> Is there a reason to add every single one?

I don't know what criteria we want to apply for exclusion/inclusion. So far it seems to have been arbitrary. 
We do need every npm/typescript and go module. We could write a script that maintains exclusions. Or maybe I can ask leeway what is a go resp. typescript project?

**Comment by svenefftinge** [comment]
I have changed this so that we only include folders that contain a `package.json` or a `go.mod` file.

File: gitpod-ws.code-workspace [hunk 1]
```diff
@@ -1,77 +1,398 @@
 {
-    "folders": [
```
Comment (mads-hartmann) [inline gitpod-ws.code-workspace:2]: Would be lovely with a comment here stating that this bit is automatically generated - and include how to run the script to update it.

**Review by mads-hartmann — APPROVED** [review]
Being intentional and consistent in the sorting seems like a good incremental change to me. Left one tiny comment.

**Pull request merged**